packages/plugins/components/src/Utils/constants.ts

```diff
@@ -0,0 +1,19 @@
+import { InterpreterOptions } from 'xstate';
```

Comment:
maybe personal preference, but open to hear you reasoning - I feel like `Utils` folder should be lowercase `utils` like `hooks`